src/Libraries/RevitNodes/Elements/InternalUtilities/ElementQueries.cs

```diff
@@ -95,34 +88,32 @@ filtering to get our intended element set.
             if (ClassFilterExceptions.Contains(elementType))
             {
                 var type = GetClassFilterExceptionsValidType(elementType);
-                return new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                return new DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
                     .OfClass(type)
                     .Where(x => x.GetType() == elementType)
-                    .Select(x => ElementSelector.ByElementId(x.Id.Value))
+                    .Select(e => e.ToDSType(true))
                     .ToList();
             }
 
-            var classFilter = new Autodesk.Revit.DB.ElementClassFilter(elementType);
-            return new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+            var classFilter = new DB.ElementClassFilter(elementType);
+            return new DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
                 .WherePasses(classFilter)
-                .ToElementIds()
-                .Select(x => ElementSelector.ByElementId(x.Value))
+                .Select(e => e.ToDSType(true))
```

Comment:
If the validation is done once per function, there shouldn't be a need to do it twice, just mentioning that at least one of these changes should be OK regardless of testing error propagation (see previous comment).